docs/articles/voice/azure-speech.md

```diff
@@ -0,0 +1,134 @@
+# DSharpPlus + Azure Speech Services
+
+## 1. Introduction
+
+With a little bit of effort, it is entirely possible to leverage Microsoft Azure's Cognitive Services for your Discord bot.  Of particular interest are the speech services, which will allow you to do [text-to-speech](https://docs.microsoft.com/en-us/azure/cognitive-services/speech-service/text-to-speech) (the focus of this article) and [speech-to-text](https://docs.microsoft.com/en-us/azure/cognitive-services/speech-service/speech-to-text).  This article will show you how to create a command to have your bot say something back to you, like Simon says.  Once you've got that under your belt, you can make modifications to have your bot speak to you based on whatever trigger and content your heart desires.  If you are feeling ambitious, you can even have your bot respond to voice messages using speech-to-text.  Just parse the returned text as if it were a regular Channel message or pass it through your Interactivity module for some real fun.
+
+## 2. Prerequisites
+
+* DSharpPlus >= 4.0
+* DSharpPlus.VoiceNext
+  * This article assumes that you've already followed the [VoiceNext](https://dsharpplus.emzi0767.com/articles/vnext_setup.html) article and have a bot that works and that can play music or some other audio.
+* NAudio [[GitHub]](https://github.com/naudio/NAudio) [[NuGet]](https://www.nuget.org/packages/NAudio/)
+* An account for [Microsoft Azure](https://portal.azure.com)
+
+## 3. Text-to-Speech
+
+Jumping right in, the process is as follows:
+
+1. Follow along with [Try Speech Services for free](https://docs.microsoft.com/en-us/azure/cognitive-services/speech-service/get-started) to get an API key for Speech Services.  The Free Tier pricing includes up to 5M characters free per month.  After that, it is $4 per 1M characters.  Not bad, really.
+
```

Comment:
Same deal with lists. Don't space between them, and for each new line add enough spaces before. E.g.

```md
1. Very long 
   list line
2. Another very 
   long line
```